### ISSUE TITLE
Sparserestore ios 18

### ISSUE THREAD

**Issue by KirChi123** [leminlimez/Nugget#296]
Does spasrserestore really work on all ios 18 version up to 18.1.1 or does it only fully work on ios 18.0-18.0.1 and partially on 18.1 and 18.1.1 and whats changed to it? What can it do more on 18.0 and 18.0.1?

**Comment by hxhlb** [comment]
As of now, the highest version that can be used is **iOS18.2 beta2**

But, please reading #206

**Comment by KirChi123** [comment]
> As of now, the highest version that can be used is **iOS18.2 beta2**
> 
> But, please reading #206

Yes i read it and i understand that the highest is 18.2 beta 2 but i thought there is a difference in the support between 18.0-18.0.1 and 18.1 and 18.2 beta 2 since apple patched the sparserestore kind of in ios 18.1 beta 4 so my question was is the sparserestore more powerful on ios 18.0-18.0.1 than 18.1-18.2 beta2?

**Comment by moon-cacke** [comment]
> > As of now, the highest version that can be used is **iOS18.2 beta2**
> > But, please reading #206
> 
> Yes i read it and i understand that the highest is 18.2 beta 2 but i thought there is a difference in the support between 18.0-18.0.1 and 18.1 and 18.2 beta 2 since apple patched the sparserestore kind of in ios 18.1 beta 4 so my question was is the sparserestore more powerful on ios 18.0-18.0.1 than 18.1-18.2 beta2?

Not sure if its different regarding the support here is what I know 

- Mobilegestalt modification work the same on 18.0.1 vs 18.1.1 (sparserestore)
- iOS v18.1.1 patched sparserestore BUT not for all locations mobilegestalt left vulnerable thus modifiable.
- v18.1.1 bring huge web exploit fix and is considered an "urgent" update. This is why I guess Apple did bot enforce sparserestore fix fully on that version
- on v18.1.1 some location for sparserestore are protected and thus future modifications/feature may be supported on 18.0.1 but not on 18.1.1 but does not look like its the case for the moment (everything works but not everything was extensively tested... its why the YMMV)


While 18.1.1 bring huge exploit fix, for me 18.0.1 had less bugs, I am facing multiple small bugs on 18.1.1 like standby not working and other tiny glitches which apparently have been fixed on 18.2

18.0.1 : full support, full support for potential future features and stable ios version BUT have a big web/app vulnerability

18.1.1 : full support and not sure about future features, ios version may have some small bugs but is a lot secure compared to older versions 

18.2 : sparse restore have been enforced and mobilegestalt is not modifiable any more (I guess because of the abuse of AI feature which costs money to Apple) the ios version can be considered the best version... support by nugget is very limited, only daemon modification and springboard works

personally I updated 18.1.1 because of the vulnerability of 18.0.1 and I am using island + mobilegestalt which works as good as on 18.0.1 
I am facing some bugs tho on 18.1.1 but for me its worth it because the version is more secur

An other good solution is 18.0.1 with some protection like adguard/else + install only reputable known apps + lockdown mode... in someway I may had preferred that to avoid those annoyances

At the end of the day there is no perfect ios version choose according to whats important to you
